plugins/inputs/prometheus/prometheus.go

```diff
@@ -75,7 +75,8 @@ type Prometheus struct {
 
 	HTTPHeaders map[string]string `toml:"http_headers"`
 
-	ResponseTimeout config.Duration `toml:"response_timeout" deprecated:"1.26.0;use 'timeout' instead"`
+	ResponseTimeout    config.Duration `toml:"response_timeout" deprecated:"1.26.0;use 'timeout' instead"`
+	ContentLengthLimit int64           `toml:"content_length_limit"`
```

Comment:
Please use `config.Size` to allow human readable sizes like `16KB`...

Reply:
Thank you. Fixed! https://github.com/influxdata/telegraf/pull/14661/commits/89c595cc3636435fa25da2fb935badd2c13129a8